app/src/main/java/org/listenbrainz/android/ui/screens/brainzplayer/BrainzPlayerBackDropScreen.kt

```diff
@@ -118,11 +121,21 @@ fun BrainzPlayerBackDropScreen(
         mutableFloatStateOf(0F)
     }
     val repeatMode by brainzPlayerViewModel.repeatMode.collectAsStateWithLifecycle()
+    val context = LocalContext.current
+    val defaultBackgroundColor = MaterialTheme.colorScheme.background
+    val isSystemInDarkTheme = isSystemInDarkTheme()
```

Comment:
We have a custom theme preference, which is different from what this function returns here. You can get the exact ui mode by using  `LocalUIMode.current`. There is also another function called `getOnScreenUiMode` or something similar that handles this case. Can you re-evaluate the usages of this variable and function as well?